caas/broker.go

```diff
@@ -109,6 +109,9 @@ type Broker interface {
 	// EnsureService creates or updates a service for pods with the given params.
 	EnsureService(appName string, params *ServiceParams, numUnits int, config application.ConfigAttributes) error
 
+	// EnsureCrd creates or updates a custom resource definition resource.
+	EnsureCrd(appName string, podSpec *PodSpec) error
```

Comment:
There seems to be a mix of "Crd" and "CustomResourceDefinition" (here and elsewhere).
k8s itself seems to prefer "CustomResourceDefinition" in it's yaml tags etc. I know it's longer but I think it reads better?